In [3]:
import pandas as pd

In [4]:
capacity = pd.read_csv("CAPACITY.csv", sep=";")

In [6]:
capacity.sample(5)

TIME_SLOTS    3G    4G    5G
2           2   750  2000  2750
5           5   500  1750  3250
1           1  1000  2000  2500
4           4   500  1750  3250
0           0  1000  2000  2500

capacité totale disponible pour chaque technologie, en fonction de ces TIME_SLOTS.

Au fil du temps, la capacité disponible pour la nouvelle génération (5G) augmente, tandis que celle des anciennes technologies (3G/4G) diminue.

CAPA NG = 750+2000+2750 = 5500